pages/wiki/building-asteroidos.hbs

```diff
@@ -10,35 +10,91 @@ layout: documentation
 <div class="page-header">
-  <h1 id="building">Building</h1>
+  <h2 id="building">Building</h2>
 </div>
 <p>Once the environment is prepared, you can simply trigger a build with the following command:</p>
 <pre><code>bitbake asteroid-image</code></pre>
-<p><i>Note:</i> Bitbake is a powerful tool that can also build single packages (e.g: bitbake strace) or <a href="{{rel 'wiki/building-asteroidos'}}">the SDK</a> for example. Refer to its documentation for more details.</p>
+<p><em>Note:</em> Bitbake is a powerful tool that can also build single packages (e.g: bitbake strace) or <a href="{{rel 'wiki/building-asteroidos'}}">the SDK</a> for example. Refer to its documentation for more details.</p>
+
+
+<div class="page-header">
+  <h1 id="build-with-docker">Build with Docker</h1>
+</div>
+
+<div class="page-header">
+  <h2 id="docker-build-setup">Setup</h2>
+</div>
+
+<p>First install Docker and Git on your system if you haven't already, then clone the main repository using the following command:</p>
+<pre><code>git clone https://github.com/AsteroidOS/asteroid.git
+cd asteroid/</code></pre>
+
+<p>If you haven't set a global git name and email yet, please do so, otherwise the build will not succeed! For example:</p>
+
+<pre><code>git config --global user.email "you@example.com"
+git config --global user.name "Your Name"</code></pre>
+
+<p>Build a Docker image called <em>asteroidos-toolchain</em> from the given Dockerfile:</p>
+<pre><code>sudo docker build --tag asteroidos-toolchain .</code></pre>
+
+<p>Remove the Docker container called <em>asteroidos-toolchain</em> in case it already exists:</p>
+<pre><code>sudo docker rm -f asteroidos-toolchain</code></pre>
+
+<div class="page-header">
+  <h2 id="docker-build-usage">Usage example</h2>
+</div>
+<p>Create a Docker container called <em>asteroidos-toolchain</em> from the Docker image which we also called <em>asteroidos-toolchain</em> and build AsteroidOS for <em>dory</em> (the LG G Watch) within the container:</p>
+<pre><code>sudo docker rm -f asteroidos-toolchain ; sudo docker run --name asteroidos-toolchain -it -v /etc/passwd:/etc/passwd -u `id -u`:`id -g` -v "$HOME/.gitconfig:/$HOME/.gitconfig" -v "$(pwd):/asteroid" asteroidos-toolchain bash -c "source ./prepare-build.sh dory && bitbake asteroid-image"</code></pre>
+
+<p>Note: The files created during the build simply go to the current direcotry (which should be <em>asteroid</em>), more spcifically, the output goes to the subdirectories <em>asteroid/src</em> and <em>asteroid/build</em>.</p>
+<p>To build for a different watch than the LG G Watch, use its corresponding codename instead of <em>dory</em> when executing the <em>docker run ...</em> command.</p>
+<p>You can find the codenames for the supported watched on the <a href="{{rel 'install'}}">Install page</a>.</p>
+
+
+<div class="page-header">
+  <h3 id="docker-run-explained">What exactly does that long <em>docker rm ... ; docker run ...</em> command do?</h3>
```

Comment:
I don't think we should have a separate section for that, you can just detail it in the above section
+ the wording of "What exactly does that long command do?" is not the wording I would expect to find in a documentation

Reply:
Okay, sure I'll change it. What did you have in mind instead of `What exactly does that long <em>docker rm ... ; docker run ...</em> command do?`?  
(It sounds pretty okay to me to be honest.)

Would `Explanation of the above <em>docker rm ... ; docker run ...</em> command:` be better?

Reply:
Yes it would be better